extras/edit_thresholds/edit_thresholds_aux.R

```diff
@@ -0,0 +1,95 @@
+library(dplyr)
+library(tidyverse)
+library(stringr)
+
+# Run together with edit_thresholds.R
+
+# Pivot longer the csv file, using the check names
+# where columns are <check_name>, <check_name>Threshold, or <check_name>Notes
+pivot_longer_func <- function(file){
+
+  df <- read.csv(file, colClasses = "character", 
+                 stringsAsFactors = FALSE)
+  
+  df_long <- df %>%
+    # Add "_" to help pivot_longer
+    rename_with(~ gsub("Threshold", "_Threshold", .x, fixed = TRUE)) %>%
+    rename_with(~ gsub("Notes", "_Notes", .x, fixed = TRUE)) %>%
+    
+    # bring all (threshold and notes) to a temp column
+    pivot_longer(-(!ends_with("Threshold") & !ends_with("Notes")),
+                 names_to = c("Check_name", "Type"),
```

Comment:
To make consistent with other column names
```suggestion
                 names_to = c("checkName", "Type"),
```

Reply:
(and now I look at it, I think I proposed `Check_name` in my example... sorry!)

Reply:
np, done